mautic/api-library
# Add DDEV first-run experience ✨

How to test:

- Checkout this branch locally
- Run `ddev start`
- The first-run experience should start 🚀 try answering "yes" so that a local Mautic instance is created
